openhands/core/cli.py

```diff
@@ -76,14 +76,35 @@ async def main():
         help='Show the version number and exit',
         default=None,
     )
+    # Add the launch subcommand
+    subparsers = parser.add_subparsers(dest='command', help='Available commands')
+    launch_parser = subparsers.add_parser('launch', help='Launch OpenHands')
+    launch_parser.add_argument('mode', choices=['cli', 'ui'], help='Launch mode')
+
     args = parser.parse_args()
 
     if args.version:
         print(f'OpenHands version: {__version__}')
         return
 
-    logger.setLevel(logging.WARNING)
-    config = load_app_config(config_file=args.config_file)
+    if args.command == 'launch':
+        directory = os.getcwd()
```

Comment:
I think this should be:
`        directory = args.directory if args.directory else os.getcwd()`